examples/miscellaneous/miscellaneous_ex12/miscellaneous_ex12.C

```diff
@@ -89,6 +90,12 @@ int main (int argc, char ** argv)
   libmesh_example_requires(false, "--enable-eigen");
 #endif
 
+  // Read the "distributed_load" flag from the command line
+  GetPot command_line (argc, argv);
+  int distributed_load = 0;
+  if (command_line.search(1, "-distributed_load"))
+    distributed_load = command_line.next(distributed_load);
+
   // Create a mesh distributed across the default MPI communicator.
   Mesh mesh (init.comm(), 3);
 
```

Comment:
Should we update all the examples to use GNU style `--long-option-names`?  I suppose it's a matter of personal preference, but at least in the library I think we do this.